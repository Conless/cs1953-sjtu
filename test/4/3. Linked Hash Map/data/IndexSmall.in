1
8 1 10 
0 3 IP8u6HGxhQAXLctE3aUz 
1 3 IP8u6HGxhQAXLctE3aUz 
0 0 vDBJ9kqNzn4aoZxgp3my 
2 0 
0 5 jxiUHRYsXOcPhVBodQ4m 
0 7 NPhBMm1KGQiEbYgdvqut 
1 5 jxiUHRYsXOcPhVBodQ4m 
0 2 FiARnVc1qxTKEQJC7Py3 
1 0 vDBJ9kqNzn4aoZxgp3my 
0 5 vMJZ5hWuyQ9aHEel8ozm 
