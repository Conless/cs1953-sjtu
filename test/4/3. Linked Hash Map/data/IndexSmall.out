vDBJ9kqNzn4aoZxgp3my 
2 FiARnVc1qxTKEQJC7Py3
5 vMJZ5hWuyQ9aHEel8ozm
7 NPhBMm1KGQiEbYgdvqut
